clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -6500,7 +6500,9 @@ class FreeFunctionPrinter {
   /// \param FD The function declaration to print.
   /// \param Args The arguments of the function.
   void printFreeFunctionDeclaration(const FunctionDecl *FD,
-                                    const std::string &Args) {
+                                    const std::string &Args,
+                                    std::string_view Templated = "") {
```

Comment:
It is unclear what Templated holds from its name.
```suggestion
                                    std::string_view TemplateParameters = "") {
```